listenbrainz_spark/recommendations/recommend.py

```diff
@@ -134,32 +139,31 @@ def get_recommendation_df(recording_ids_and_ratings):
     return df
 
 
-def scale_ratings(mbids_and_ratings, ratings_beyond_range):
+def scale_ratings(mbids_and_ratings, params):
     """ Scale the ratings so that they fall on a range from 0.0 -> 1.0.
 
         Args:
             mbids_and_ratings: list of recommended recording mbids and ratings.
-            ratings_beyond_range: list to update with ratings beyond expected range.
+            params: RecommendationParams class object.
```

Comment:
We should get into the habit of adding these as type annotations. That way when a person is reading the code in a text editor, they can just control+click to go to the  definition of the class.

Reply:
Something like https://github.com/metabrainz/listenbrainz-server/pull/1062#discussion_r479773303